SU-SOE/stanford_soe_helper
# 2-Column Text Style from Injector 44 to SCSS

# READY FOR REVIEW

# Summary
This adds the Collections Injector file 44 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 44 into SCSS


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-2987 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2987)
- delete this injector: `admin/config/development/css-injector/edit/44`
- `drush cc all`
- go to: `/`, `magazine/article/researchers-build-water-based-battery-store-solar-and-wind-energy`,  and `magazine/article/researchers-analyze-friendship-networks-infer-hidden-traits`.
- make sure the css is showing.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2987

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
